music_assistant/controllers/players.py

```diff
@@ -1509,7 +1509,11 @@ def iter_group_members(
         exclude_self: bool = True,
     ) -> Iterator[Player]:
         """Get (child) players attached to a group player or syncgroup."""
-        for child_id in list(group_player.group_members):
+        group_members = group_player.group_members.copy()
+        if not exclude_self and group_player.player_id not in group_members:
+            # Always include the group player itself if not excluded
```

Comment:
this is weird... so a group player is suddenly going to be included here ?
Let's not do that.